images/installPlatform.sh

```diff
@@ -43,17 +43,14 @@ VERSION="$2"
 
 debianFlavor=$DEBIAN_FLAVOR
 fileName="$PLATFORM_NAME-$VERSION.tar.gz"
-
-sdkStorageAccountUrl="$ORYX_SDK_STORAGE_BASE_URL"
```

Comment:
We need this logic so that we can control where the pre-baked binaries come from (dev during testing, prod during actual image release)

Reply:
Looks like you already reverted this change. Thanks!